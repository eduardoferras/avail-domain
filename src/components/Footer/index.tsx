import Image from 'next/image'
import Link from 'next/link'
import logo from '@/images/logo-avail-domain.svg'
import S from './styles.module.scss'

export default function Footer() {
	return (
		<footer className={S.footer}>
			<p className={S.content}>Copyright AvailDomain 2024 - Seu consultor de Dom√≠nios</p>
		</footer>
	)
}
