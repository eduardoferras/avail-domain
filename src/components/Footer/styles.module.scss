.footer {
	padding: 1.6rem 0;
	background-color: #fff;
	border-top: 1px solid #A4A4A4;

	&__content {
		@include container;
		color: #4F6066;
		font-family: var(--font-primary);
		font-size: 1.4rem;
		font-style: normal;
		font-weight: 400;
		line-height: normal;
		text-align: center;
	}
}
