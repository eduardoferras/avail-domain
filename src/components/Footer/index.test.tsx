import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Footer from '.'

describe('Footer', () => {
	it('renders a paragraph', () => {
		renderFooter()

		const paragraph = screen.getByRole('paragraph')

		expect(paragraph).toBeInTheDocument()
	})

	it('should be rendered correctly', () => {
		const { container } = renderFooter()

		expect(container).toMatchSnapshot()
	})
})

function renderFooter() {
	return render(<Footer />)
}
