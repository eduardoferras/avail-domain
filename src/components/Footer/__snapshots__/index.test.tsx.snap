// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should be rendered correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <p
      class="footer__content"
    >
      Copyright AvailDomain 2024 - Seu consultor de Dom√≠nios
    </p>
  </footer>
</div>
`;
